chore: tidy Go prereqs and guide (#667)
wasmCloud/wasmCloud.com: docs/tour/hello-world.mdx

scoop install binaryen
```

> You will also need to [download the binary for the latest version of `wasm-tools` for your architecture](https://github.com/bytecodealliance/wasm-tools#installation) and add it to your PATH.

Reviewer: ```suggestion
You will also need to [install the latest version of `wasm-tools`](https://github.com/bytecodealliance/wasm-tools#installation) and add it to your PATH.
```

curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```

> Once you've installed Rust on macOS or Linux, use `rustup` to add the `wasm32-wasip1` target:

Reviewer: ```suggestion
Once you've installed the Rust toolchain, use `rustup` to add the `wasm32-wasip1` target:
```

Generally `rustup` works on windows the same way right *once it's installed*?

Might also be worth including a link to the [windows install docs](https://forge.rust-lang.org/infra/other-installation-methods.html) earlier on (i.e. *before* Rust is installed)... 

Maybe something like:

```md
On Windows, the Rust project providers [standalone installers](https://forge.rust-lang.org/infra/other-installation-methods.html) as an alternate installation method
```

Note: On Windows, you will also need Visual Studio's C++ tools.

> Once you've installed Rust on Windows, use `rustup` to add the `wasm32-wasip1` target:

Reviewer: ```suggestion
Once you've installed Rust, use `rustup` to add the `wasm32-wasip1` target:
```
